components/your-history/ActivityCard.js

```diff
@@ -0,0 +1,144 @@
+  },
+  chip: {
+    marginRight: theme.spacing(1),
+  },
+  header: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'flex-start',
+  },
+  description: {
+    marginTop: theme.spacing(4),
+    marginBottom: theme.spacing(4),
+  },
+}));
+
+function ActivityCard(props) {
+  const classes = useStyles();
+
+  const {
+    timestamp,
+    activityTypeLabel,
+    description,
+    briefDescription,
+    dateCompleted,
+    timeSpentInMinutes,
+    difficultyLevel,
+    activityFeeling,
+    whyIfeelThisWay,
+  } = props;
+  return (
+    <Card className={classes.card}>
+      <CardContent>
+        <div className={classes.header}>
+          <Typography variant="body1" noWrap style={{ maxWidth: '50%', fontWeight: 500 }}>
+            {activityTypeLabel}
+          </Typography>
+          <Typography variant="caption" component="p" noWrap>
+            {getFormattedDateEntered(timestamp)}
+          </Typography>
+        </div>
+
+        <Typography variant="h6" component="h2" className={classes.group}>
+          {briefDescription}
+        </Typography>
+        <Typography variant="body2" component="p" className={classes.description}>
+          {description}
+        </Typography>
+        <Grid container className={classes.group}>
+          <Grid item xs={4}>
+            <Typography variant="body2" className={classes.label}>
+              Completed On
+            </Typography>
+            <Typography variant="body1">{getFormattedDateCompleted(dateCompleted)}</Typography>
+          </Grid>
+          <Grid item xs={4}>
+            <Typography variant="body2" className={classes.label}>
+              Difficulty
+            </Typography>
+            <Typography variant="body1">{difficultyLevel}</Typography>
+          </Grid>
+          <Grid item xs={4}>
+            <Typography variant="body2" className={classes.label}>
+              Estimated Time
+            </Typography>
+            <Typography variant="body1" component="span" display="block">
+              {timeSpentInMinutes} Minutes
+            </Typography>
+          </Grid>
+        </Grid>
+        <Divider className={classes.divider} />
+        <div className={classes.group}>
+          <Typography variant="body2" className={classes.label} gutterBottom>
+            I felt...
+          </Typography>
+          <Grid>
+            {activityFeeling.map(feeling => (
+              <Chip key={feeling} label={feeling} className={classes.chip} />
+            ))}
+          </Grid>
+        </div>
+        <div className={classes.group}>
+          <Typography variant="body2" className={classes.label} gutterBottom>
+            I felt like this because...
+          </Typography>
+          <Typography variant="body2" component="p">
+            {whyIfeelThisWay}
+          </Typography>
+        </div>
+      </CardContent>
+    </Card>
+  );
+}
+
+ActivityCard.propTypes = ActivityPropTypes;
+
+ActivityCard.defaultProps = {
```

Comment:
Feels odd having `defaultProps` in the file and `propTypes` external -- is there a reason for this?

Reply:
(I might be unaware of a best practice here)

Reply:
moved it to activity card for now